File: SistemaVentaAngular/Models/VentasCredito.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace SistemaVentaAngular.Models
{
    public partial class VentasCredito
    {
        public VentasCredito()
        {
            DetalleVentasCredito = new HashSet<DetalleVentasCredito>();
        }

        public int IdVentasCredito { get; set; }
        public string? NumeroDocumento { get; set; }
        public string? TipoPago { get; set; }
        public DateTime? FechaRegistro { get; set; }
        public decimal? Total { get; set; }
        public bool IsPaid { get; set; }
        [NotMapped]
        public string? customerName { get; set; }
        public bool isDelete { get; set; }
        public virtual ICollection<DetalleVentasCredito> DetalleVentasCredito { get; set; }

    }
}

File: SistemaVentaAngular/ClientApp/src/app/interfaces/ventas-credito.ts
import { DetalleVentasCredito } from "./detalle-ventas-credito";

export interface VentasCredito {
  idVenta?: number,
  numeroDocumento?: string,
  fechaRegistro?: string,
  cantidadML: string,
  tipoPago?: string,
  totalTexto?: string,
  detalleVentasCredito?: DetalleVentasCredito[]
}
